DxDispatch/src/dxdispatch/dxDispatchWrapper.cpp

```diff
@@ -188,6 +189,41 @@ HRESULT DxDispatch::RuntimeClassInitialize(
         m_device->ClearShaderCaches();
     }
 
+    if (m_options->DisableBackgroundProcessing())
+    {
+        HRESULT hr = m_device->D3D()->SetBackgroundProcessingMode(
+            D3D12_BACKGROUND_PROCESSING_MODE_DISABLE_BACKGROUND_WORK,
+            D3D12_MEASUREMENTS_ACTION_KEEP_ALL,
+            nullptr,
+            nullptr
+        );
+
+        if (FAILED(hr))
+        {
+            m_logger->LogError("Failed to disable background processing. Do you have developer mode enabled?");
+            return hr;
+        }
+        else
+        {
+            m_logger->LogInfo("Background processing DISABLED");
+        }
+    }
+
+    if (m_options->SetStablePowerState())
+    {
+        // TODO: smart object to unset state
+        HRESULT hr = m_device->D3D()->SetStablePowerState(TRUE);
```

Comment:
>HRESULT h [](http://example.com/codeflow?start=8&length=9)

I would add the comment from msdn

[https://learn.microsoft.com/en-us/windows/win32/api/d3d12/nf-d3d12-id3d12device-setstablepowerstate](https://learn.microsoft.com/en-us/windows/win32/api/d3d12/nf-d3d12-id3d12device-setstablepowerstate)

"Do not call this method in normal execution for a shipped application. This method only works while the machine is in developer mode. If developer mode is not enabled, then device removal will occur. Instead, call this method in response to an off-by-default, developer-facing switch. Calling it in response to command line parameters, config files, registry keys, and developer console commands are reasonable usage scenarios."

Reply:
I added the dev mode requirement to the command line options. Together with the error message, I think it should be pretty clear, but I don't expect most people to use this outside of experts in D3D12. What do you think?

Reply:
I think that addresses the comment, if someone uses code as a template I just wanted to be sure it is clear this added code is only for development bring up not added to regular apps.